src/OSPSuite.Infrastructure.Export/ExportToExcelTask.cs

```diff
@@ -58,37 +76,22 @@ private static void saveWorkbook(string fileName, XSSFWorkbook workBook)
          });
       }
 
-      private static void exportDataTableToWorkBook(IWorkbookConfiguration workBookConfiguration, DataTable dataTable)
+      private static void exportDataTableToWorkBook(DataTable dataTable, IWorkbookConfiguration workBookConfiguration)
       {
-         // Thread.CurrentThread.CurrentCulture.NumberFormat.NumberDecimalSeparator = ".";
-
          var workBook = workBookConfiguration.WorkBook;
-
-         ISheet sheet;
-
-         if (!dataTable.TableName.Equals(""))
-         {
-            sheet = workBook.CreateSheet(dataTable.TableName);
-         }
-         else
-         {
-            sheet = workBook.CreateSheet("Sheet1");
```

Comment:
this was wrong here. Exporting to DataTable with names would crash. It needs to be done before exporting ONE single sheet

Reply:
wow, means it crashed like almost all the time before? wtf

Reply:
no no. Only if not table names were given and you had at least 2 tables. SO not so often (not sure if this is maybe just a theoretical problem)